apps/web/components/FrontpageSlider/LottiePlayer/react-lottie/useLottie.tsx

```diff
@@ -0,0 +1,312 @@
+import React, {
```

Comment:
Shouldn't this be a lib?

if not in the repo lib root, then in `apps/web/libs/...`

Reply:
Good call